lib/glific_web/resolvers/tickets.ex

```diff
@@ -83,4 +83,17 @@ defmodule GlificWeb.Resolvers.Tickets do
       {:ok, %{success: true, message: "Updated successfully"}}
     end
   end
+
+  @doc """
+  close multiple tickets on the basis of topic
+  """
+  @spec update_ticket_status_based_on_topic(Absinthe.Resolution.t(), %{input: map()}, %{
+          context: map()
+        }) ::
+          {:ok, map()} | {:error, map()}
+  def update_ticket_status_based_on_topic(_, %{input: params}, _) do
+    with true <- Tickets.update_ticket_status_based_on_topic(params) do
+      {:ok, %{success: true, message: "Updated successfully"}}
+    end
```

Comment:
Why do we need with when the function response is always true?

Reply:
yaa, I think this is unnecessary I'll just remove this